HTW-GT4-WS24-25/TobiasFalke
# Player movement enhancements

This PR includes the implementation of a skateboard sprite for the player prefab and a first implementation of the jump.

# How do I test this?

Just start the game, play around and see if you like how the jump and skateboard feel at the moment.
